### ISSUE TITLE
[Bug]: mo-panic for mpool freed more bytes than alloc when run bvt test

### ISSUE THREAD

**Issue by aressu1985** [matrixorigin/matrixone#6138]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):554cca326583c75b4c438c2888345e810c76b655
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

when run bvt the case ：
[ERROR]
[SCRIPT   FILE]: /data1/sudong/mo-regression/matrixone/test/cases/dml/select/minus.sql
[ROW    NUMBER]: 80
[SQL STATEMENT]: create table t4(
col1 float,
col2 decimal(5,2)
);
[EXPECT RESULT]:
null
[ACTUAL RESULT]:
internal error: mpool freed more bytes than alloc

mo painc and crash , the log：

2022/10/31 22:12:32.176839 +0800 INFO mpool/mpool.go:97 error: internal error: mpool freed more bytes than alloc {"span": {}}
2022/10/31 22:12:32.176435 +0800 INFO mpool/mpool.go:97 error: internal error: mpool freed more bytes than alloc {"span": {}}
2022/10/31 22:12:32.175578 +0800 ERROR mpool/mpool.go:96 Mpool session free bug, stats:      allocations : 602526
     frees : 611803
     go runtime allocations : 0
     alloc bytes : 11594669973
     free bytes : 11879767929
     current bytes : -285097956
     high water mark : 8799459637

github.com/matrixorigin/matrixone/pkg/common/mpool.(*MPoolStats).RecordFree
        /data1/sudong/mo-regression/matrixone/pkg/common/mpool/mpool.go:96
github.com/matrixorigin/matrixone/pkg/common/mpool.(*MPool).Free
        /data1/sudong/mo-regression/matrixone/pkg/common/mpool/mpool.go:563
github.com/matrixorigin/matrixone/pkg/container/hashtable.(*StringHashMap).Free
        /data1/sudong/mo-regression/matrixone/pkg/container/hashtable/string_hash_map.go:53
github.com/matrixorigin/matrixone/pkg/common/hashmap.(*StrHashMap).Free
        /data1/sudong/mo-regression/matrixone/pkg/common/hashmap/strhashmap.go:69
github.com/matrixorigin/matrixone/pkg/sql/colexec/minus.Call
        /data1/sudong/mo-regression/matrixone/pkg/sql/colexec/minus/minus.go:87
github.com/matrixorigin/matrixone/pkg/vm.Run
        /data1/sudong/mo-regression/matrixone/pkg/vm/vm.go:61
github.com/matrixorigin/matrixone/pkg/vm/pipeline.cleanup
        /data1/sudong/mo-regression/matrixone/pkg/vm/pipeline/pipeline.go:155
github.com/matrixorigin/matrixone/pkg/vm/pipeline.(*Pipeline).MergeRun.func1
        /data1/sudong/mo-regression/matrixone/pkg/vm/pipeline/pipeline.go:125
github.com/matrixorigin/matrixone/pkg/vm/pipeline.(*Pipeline).MergeRun

panic: internal error: mpool freemany freed more bytes than alloc

goroutine 7056081 [running]:
github.com/matrixorigin/matrixone/pkg/common/mpool.(*MPoolStats).RecordManyFrees(0xca8aa81c08?, {0x252aa78, 0x7}, 0x22a5ea0?, 0xc78c39d570?)
        /data1/sudong/mo-regression/matrixone/pkg/common/mpool/mpool.go:108 +0x10c
github.com/matrixorigin/matrixone/pkg/common/mpool.(*MPool).destroy(0x3c74200?)
        /data1/sudong/mo-regression/matrixone/pkg/common/mpool/mpool.go:332 +0x9d
github.com/matrixorigin/matrixone/pkg/common/mpool.DeleteMPool(0xc028d3f180)
        /data1/sudong/mo-regression/matrixone/pkg/common/mpool/mpool.go:428 +0x48
github.com/matrixorigin/matrixone/pkg/frontend.(*Session).Dispose(0x47?)
        /data1/sudong/mo-regression/matrixone/pkg/frontend/session.go:147 +0x28
github.com/matrixorigin/matrixone/pkg/frontend.(*Routine).Quit.func1()
        /data1/sudong/mo-regression/matrixone/pkg/frontend/routine.go:184 +0x2f
sync.(*Once).doSlow(0x0?, 0xc00330c480?)
        /usr/local/go/src/sync/once.go:74 +0xc2
sync.(*Once).Do(...)
        /usr/local/go/src/sync/once.go:65
github.com/matrixorigin/matrixone/pkg/frontend.(*Routine).Quit(0x254707a?)
        /data1/sudong/mo-regression/matrixone/pkg/frontend/routine.go:181 +0x45
github.com/matrixorigin/matrixone/pkg/frontend.(*RoutineManager).Closed(0xc008a10800, {0x29369a0, 0xc014741100})
        /data1/sudong/mo-regression/matrixone/pkg/frontend/routine_manager.go:133 +0x12e
github.com/fagongzi/goetty/v2.(*baseIO).Close(0xc005c80e10?)
        /home/sudong/go/pkg/mod/github.com/fagongzi/goetty/v2@v2.0.3-0.20221026091852-7f85e50d3c1f/session.go:370 +0x1a2
github.com/fagongzi/goetty/v2.(*server).doStart.func2.1()
        /home/sudong/go/pkg/mod/github.com/fagongzi/goetty/v2@v2.0.3-0.20221026091852-7f85e50d3c1f/application.go:296 +0x5e
github.com/fagongzi/goetty/v2.(*server).doStart.func2()
        /home/sudong/go/pkg/mod/github.com/fagongzi/goetty/v2@v2.0.3-0.20221026091852-7f85e50d3c1f/application.go:304 +0x1c3
created by github.com/fagongzi/goetty/v2.(*server).doStart
        /home/sudong/go/pkg/mod/github.com/fagongzi/goetty/v2@v2.0.3-0.20221026091852-7f85e50d3c1f/application.go:293 +0x4bb

**Comment by m-schen** [comment]
which .toml file was used.
Didn't ci repo it?

**Comment by m-schen** [comment]
mpool bug was fixed by commit 244c0ca0a4422ccc453a15c2b0b85e5dfd3f8ade.
maybe we will have an elegant way to fix it next day.
but it's fixed now.

**Comment by aressu1985** [comment]
OK, this issue will be tested with #6190 tomorrow

**Comment by aressu1985** [comment]
testing

**Comment by aressu1985** [comment]
fixed:622d40ed44fb8b745048f22629e4553da127731e
